examples/post_training_quantization/torch/mobilenet_v2/main.py

```diff
@@ -0,0 +1,179 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import re
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import nncf
+import numpy as np
+import openvino.runtime as ov
+import torch
+from sklearn.metrics import accuracy_score
+from openvino.offline_transformations import compress_quantize_weights_transformation
+from torchvision import datasets
+from torchvision import models
+from torchvision import transforms
+from tqdm import tqdm
+
+FOOD101_CLASSES = 101
+ROOT = Path(__file__).parent.resolve()
+DATASET_PATH = Path(sys.argv[1]) if len(sys.argv) > 1 else ROOT / 'dataset'
+CHECKPOINT_URL = 'https://huggingface.co/AlexKoff88/mobilenet_v2_food101/resolve/main/pytorch_model.bin'
+
+
+def fix_names(state_dict):
+    state_dict = {key.replace('module.', ''): value for (key, value) in state_dict.items()}
+    return state_dict
+
+
+def load_checkpoint(model):  
+    checkpoint = torch.hub.load_state_dict_from_url(CHECKPOINT_URL, progress=False)
+    weights = fix_names(checkpoint['state_dict'])
+    model.load_state_dict(weights)
+    return model
+
+
+def validate(model, val_loader):
+    predictions = []
+    references = []
+
+    output_name = model.outputs[0]
+
+    for images, target in tqdm(val_loader):
+        pred = model(images)[output_name]
+        predictions.append(np.argmax(pred, axis=1))
+        references.append(target)
+
+    predictions = np.concatenate(predictions, axis=0)
+    references = np.concatenate(references, axis=0)  
+    return accuracy_score(predictions, references)
+
+
+def ov_convert(model, args):
+    onnx_model_path = f'{tempfile.gettempdir()}/model.onnx'
```

Comment:
I think it is better to use such approach
```python
with tempfile.TemporaryDirectory(dir=tempfile.gettempdir()) as temp_dir:
    ...
```
because `temp_dir` will be removed after exiting from the `with` statement.

Reply:
The function was removed.

Reply:
There is another place where it can be fixed

https://github.com/alexsu52/nncf/blob/9acf21d6cedb15487e1eaf79731039f630082995/examples/post_training_quantization/torch/mobilenet_v2/main.py#L70-L72